src/GitLinkTask/GitLink.cs

```diff
@@ -18,21 +18,33 @@ public class GitLink : Task
         [Required]
         public ITaskItem PdbFile { get; set; }
 
-        public bool DownloadWithPowershell { get; set; }
+        public string Method
+        {
+            get { return this.MethodEnum.ToString(); }
```

Comment:
Get rid of the this. please (but probably already done that after feedback on previous PR)